dev-python/soundfile/soundfile-0.12.1.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_EXT=1
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( pypy3 python3_{10..13} )
+
+inherit distutils-r1
+
+DESCRIPTION="SoundFile is an audio library based on libsndfile, CFFI, and NumPy"
+HOMEPAGE="
+	https://github.com/bastibe/python-soundfile/
+"
+SRC_URI="
+	https://github.com/bastibe/python-soundfile/archive/${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+S="${WORKDIR}/python-soundfile-${PV}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
```

Comment:
The test flag is already added by `distutils_enable_tests`

Reply:
I wanted to remove debug USE-flag. It didn't work.

Reply:
Can you clarify further? 

The `ISUE="test"` line is not needed and can be removed.

Reply:
I thought it would override list of avaliable use flags. As it turns out I should just remove DISTUTILS_EXT=1.

Reply:
Forgot this one. Force-pushing again.